odoo/tools/mail.py

```diff
@@ -176,7 +176,23 @@ def parse_style(self, el):
                 del el.attrib['style']
 
 
-def html_sanitize(src, silent=True, sanitize_tags=True, sanitize_attributes=False, sanitize_style=False, sanitize_form=True, strip_style=False, strip_classes=False):
+def html_sanitize(src, silent=True, sanitize_tags=True, sanitize_attributes=False, sanitize_style=False,
+                  sanitize_form=True, strip_style=False, strip_classes=False, sanitize_extra_tags=None):
+    """Sanitize HTML content to prevent XSS attacks or unwanted content.
+
+    This function wraps`lxml.html.clean.Cleaner` to process and sanitize HTML content according to the provided arguments.
+
+    :param str src: The HTML source string to be sanitized.
+    :param bool silent: If True, suppresses most of the errors that may occur during sanitization. If False, errors may be raised.
+    :param bool sanitize_tags: If True, sanitizes HTML tags to remove potentially dangerous tags such as <script>, <iframe>, etc.
+    :param bool sanitize_attributes: If True, filters out harmful attributes from tags that can lead to XSS attacks, such as "onclick".
+    :param bool sanitize_style: If True, cleans up CSS styles within HTML elements to prevent malicious code execution.
```

Comment:
it's not really about code execution, it's more about cleaning style attribute
```suggestion
    :param bool sanitize_style: If True, cleans up style attributes
```